unity/PitayaExample/Assets/Pitaya/Editor/PitayaXcodeProjectUpdater.cs

```diff
@@ -21,8 +26,31 @@ public static void OnPostprocessBuild(BuildTarget buildTarget, string path)
 			// Pitaya should be linked with zlib when on iOS.
 			proj.AddBuildProperty(targetGuid, "OTHER_LDFLAGS", "-lz");
 
+
+			foreach (string file in Directory.GetFiles(GetPathToHide(path))) {
+				string fileGuid = proj.FindFileGuidByProjectPath(file.Substring(path.Length + 1));
+				// Files will be kept in the folder, but won't appear in the Project
+				proj.RemoveFile(fileGuid);
+			}
+
 			File.WriteAllText(projPath, proj.WriteToString());
 		}
 	}
+
+	private static string GetPathToHide(string projectPath)
+	{
+		string originPath = Path.Combine(projectPath, FRAMEWORK_ORIGIN_PATH);
+		if (!Directory.Exists(originPath))
+		{
+			originPath = Path.Combine(projectPath, FRAMEWORK_NUGET_ORIGIN_PATH);
```

Comment:
Should we do anything in case this path also does not exist? I don't think we need to cover this scenario, IMO, as we're expecting only these two installation paths.

Reply:
As we saw previous, an error here doesn't prevent the build to finish, so even both folders didn't exist, the build won't fail. I don't know if logging something here could be useful, so I think we should do nothing regarding this.

Reply:
Agreed, LGTM, then!